src/main/java/it/gov/pagopa/payment/controller/CommonPaymentController.java

```diff
@@ -0,0 +1,18 @@
+package it.gov.pagopa.payment.controller;
```

Comment:
let's introduce the payment nested package also here (make this to all controller involved)
```suggestion
package it.gov.pagopa.payment.controller.payment;
```